plugin/storage/es/spanstore/reader.go

```diff
@@ -183,13 +189,17 @@ func (s *SpanReader) indicesForTimeRange(indexName string, startTime time.Time,
 
 // GetServices returns all services traced by Jaeger, ordered by frequency
 func (s *SpanReader) GetServices(ctx context.Context) ([]string, error) {
+	span, ctx := startSpanForQuery(ctx, "GetServices", serviceNameField)
+	defer span.Finish()
 	currentTime := time.Now()
 	jaegerIndices := s.indicesForTimeRange(s.serviceIndexPrefix, currentTime.Add(-s.maxSpanAge), currentTime)
 	return s.serviceOperationStorage.getServices(jaegerIndices)
 }
 
 // GetOperations returns all operations for a specific service traced by Jaeger
 func (s *SpanReader) GetOperations(ctx context.Context, service string) ([]string, error) {
+	span, ctx := startSpanForQuery(ctx, "GetOperations", operationNameField)
```

Comment:
I don't think this makes much sense in this form. In case of Cassandra storage, there was actually a CQL query that was being logger, but here you're just using constants that are already implied by the operation name. I think you can simply use `opentracing.StartSpanFromContext(ctx, "operation")`.

Since ES doesn't really have a "query", what might be useful instead is logging the index names. But that's optional.

Reply:
Basically the spans should be capturing parameters that may be helpful in debugging.

Reply:
Yes, sorry for the confusion, will clear this out. I didn't realise that `startSpanForQuery` was a wrapper you made for the particular use case until much later. I thought this was the norm.